docs/guides/variables.md

```diff
@@ -57,38 +62,40 @@ You can access any variable through the Python SDK with the `Variable.get()` met
 ```python
 from prefect.variables import Variable
 
-# setting the variable
-variable = Variable.set(name="the_answer", value="42")
-
-# getting from a synchronous context
-answer = Variable.get('the_answer')
-print(answer.value)
-# 42
-
-# getting from an asynchronous context
-answer = await Variable.get('the_answer')
-print(answer.value)
-# 42
-
-# getting without a default value
-answer = Variable.get('not_the_answer')
-print(answer.value)
-# None
-
-# getting with a default value
-answer = Variable.get('not_the_answer', default='42')
-print(answer.value)
-# 42
-
-# using `overwrite=True`
-answer = Variable.get('the_answer')
-print(answer.value)
-#42
-answer = Variable.set(name="the_answer", value="43", overwrite=True)
-print(answer.value)
-#43
+# set a variable
+Variable.set(name="the_answer", value="42")
```

Comment:
Any reason not to keep the assignment in here?
```suggestion
variable = Variable.set(name="the_answer", value="42")
```

Reply:
We changed the interface so `set` doesn't return anything but now I am reconsidering it. `get` returns the value by default and the full object optionally, going to get a couple more opinions on what "feels good" here